src/slicegpt/hf_utils.py

```diff
@@ -74,48 +74,55 @@ def wrapper(*args, **kwargs):
 
 @do_not_initialize
 def get_model_and_tokenizer(
```

Comment:
Can we add a test for loading local models? This could e.g. download OPT-125m, delete a layer then save locally, then loaded using the local model load logic. Documentation for local model loading could be useful too